app/api/request_api_utils.py

```diff
@@ -36,6 +36,7 @@ def post_request(request_string, data):
             access_expiry_cookie = response_message.get("access_expiry")
             AUTH_COOKIE["Authorization"] = f"Bearer {access_token_cookie}"
             AUTH_COOKIE["Authorization"]["expires"] = access_expiry_cookie
+            AUTH_COOKIE["user_id"] = 0
```

Comment:
why is this 0? @mtreacy002

Reply:
@isabelcosta. This is because if the server only just run for the first time, there'll be no AUTH_COOKIE["user_id"] ever initiated in the system. I tried once with `if not AUTH_COOKIE["user_id"]` to check if it's `none` but the system crash since there's no such attribute as `user_id` in the AUTH_COOKIE. But if the system already has someone logged in, their "user_id" need to be cleared out so another logged in person not going to be accidentally associated to it. I also tried to set AUTH_COOKIE["user_id"] = None, but that didn't work as well (forgot what thee error message was).
So here, I used `0` as a default on AUTH_COOKIE["user_id"] since the database will never give an index  of `0` as an ID to an instanced (it always starts with 1). So, it's sort of the safest way to instantiate the attribute "user_id" for AUTH_COOKIE but without associating it to any user.

Reply:
I think we should keep it as None. We can look into the error, it would just be a TypeError, where you can cast None to 0 if needed

Reply:
Got it. I changed it to None and use catch the ValueError exception. Perhaps that crash I encountered last time was on something else then 😊